src/home_robot/home_robot/agent/ovmm_agent/ppo_agent.py

```diff
@@ -226,7 +228,7 @@ def reset(self) -> None:
 
         self.prev_actions = torch.zeros(
             1,
-            self.num_actions,
+            self.action_shape,
```

Comment:
what if this is not an int

Reply:
It's always an int for us. For discrete/categorical actions it's 1, otherwise for continuous actions we use a 1-D vector.

Reply:
should it be called action_dim then?

Reply:
That sounds like a better name. Renamed to `actions_dim`.